tests/test_filter_count.py

```diff
@@ -750,6 +750,29 @@ def test_num_label_issues_different_estimation_types():
     assert n3 != n2
 
 
+def test_find_label_issues_same_value():
+    f1 = filter.find_label_issues(
+        labels=data["labels"],
+        pred_probs=data["pred_probs"],
+        filter_by="confident_learning",
+    )
+
+    f2 = filter.find_label_issues(
+        labels=data["labels"],
+        pred_probs=data["pred_probs"],
+        filter_by="low_self_confidence",
+    )
+
+    f3 = filter.find_label_issues(
+        labels=data["labels"],
+        pred_probs=data["pred_probs"],
+        filter_by="low_normalized_margin",
+    )
+
+    assert np.sum(f1) == np.sum(f2)
```

Comment:
FYI a similar test is done [here](https://github.com/cleanlab/cleanlab/blob/master/tests/test_filter_count.py#L1176), perhaps we can combine these two to reduce some compute?